webapp/src/index.css

```diff
@@ -13,7 +13,7 @@
   --title-font-size: 2.5rem;
   --subtitle-font-size: 2rem;
   --heading-font-size: 1.5rem;
-  --subheading-font-size: 1.25rem;
+  --subheading-font-size: 1.15rem;
```

Comment:
I think you may be looking at the wrong Figma file. The good one is this one -> https://www.figma.com/file/0es84Oh1roPphFOd2GYysn/Library?node-id=157%3A26. Despite this, you'll see that with 1.15rem is 18.4px, I have round off the decimals

That being said, please, correct me if I am wrong!!

Reply:
Oh, you are right. The main Figma file still has the old values. Is there a way to change it to use the library values?

Reply:
I was investigating it and it seems that, since that page has been done with the local lib, we have to substitute every component with the "new" library ones... :( I'd say it does not worth it. We could start using the new library with the next MVP